README.md

```diff
@@ -2,7 +2,7 @@
 
 A 12 column responsive grid system for laying out documents, templates and modules.
 
-> Living off the grid and being kind of an outlaw brings a dangerous reality.  
+> Living off the grid and being kind of an outlaw brings a dangerous reality.
```

Comment:
Oh, TIL! well that'll be fun, cause my editor won't let me save without removing trailing whitespace :)) No, I'll obviously fix that